NEWS.md

```diff
@@ -13,6 +13,8 @@
 
 * Closed #225: Added `tagInsertChildren()` to be able to insert child tag objects at a particular location. (#224)
 
+* `copyDependencyToDir()` no longer creates empty directories for dependencies that do not have any files. (@gadenbuie, #276)
```

Comment:
@gadenbuie would you mind rebasing/merging with `main` so this gets put in the right place?